dynamic-relations/src/main/java/at/drm/service/DynamicRelationsPrintService.java

```
@@ -0,0 +1,127 @@
+            TreeNodeRelationIdentity childObject;
+            for (RelationLink relation : relations) {
+                var entityClass = classBySimpleName.get(relation.getTargetType());
+                var childRelationIdentity = entityManager.find(entityClass, relation.getTargetId());
+                childObject = new TreeNodeRelationIdentity(childRelationIdentity, new LinkedList<>());
+                var r = new Relation(parentObject.object(), childObject.object());
+                if (!visited.contains(r)) {
+                    parentObject.childObjects().add(childObject);
+                    visited.add(r);
+                    stack.add(childObject);
+                }
+            }
+        }
+    }
+
+    @PostConstruct
+    public void configure() {
+        Set<Class<RelationIdentity>> classesImplementingInterface = findClassesImplementingInterface();
+        classBySimpleName = classesImplementingInterface.stream()
+            .collect(Collectors.toMap(clazz -> clazz.getSimpleName() + "Type", Function.identity()));
+    }
+
+
+    private <T> Set<Class<T>> findClassesImplementingInterface() {
+        Set<Class<T>> classes = new HashSet<>();
+
+        ClassPathScanningCandidateComponentProvider scanner =
+            new ClassPathScanningCandidateComponentProvider(false);
+
+        scanner.addIncludeFilter(new AssignableTypeFilter(RelationIdentity.class));
+        Set<BeanDefinition> candidates = scanner.findCandidateComponents("*");
+        for (BeanDefinition candidate : candidates) {
+            try {
+                Class<T> clazz = (Class<T>) Class.forName(candidate.getBeanClassName());
+                if (!clazz.isInterface() && !Modifier.isAbstract(clazz.getModifiers())) {
+                    classes.add(clazz);
+                }
+            } catch (ClassNotFoundException e) {
+                System.err.println("Class not found: " + candidate.getBeanClassName());
```

Use the log.error instead of the System.err
I think it should also throw an Runtime Exception